SalaryAppTests/SalaryAppTests.swift

```diff
@@ -12,21 +12,27 @@ final class SalaryAppTests: XCTestCase {
 
     
     let salaryVm = SalaryViewModel()
```

Comment:
Follow what i said before and create this in the setup function & dispose of it in the tear down read up on it if you dunno what i mean